src/AddTodoForm.jsx

```diff
@@ -16,11 +17,7 @@ const AddTodoForm = ({onAddTodo}) => {
     return (
         <>
             <form onSubmit={handleAddTodo}>
-                <label>
-                    Title
-                    <input value={todoTitle} onChange={handleTitleChange} name="title" id="todoTitle"/>
-                </label>
-
+                <InputWithLabel listTitle={todoTitle} handleTitleChange={handleTitleChange}>Title </InputWithLabel>
```

Comment:
We should make the prop names here generic rather than specific to "title". The goal of making this `InputWithLabel` component is that we could use it elsewhere in our site as an input for different data (maybe a password, or a username, etc.). I think names here should be "value" and "handleChange" or something similar to this, that doesn't mention a title. The value you pass to these props should be what makes this input "specific" to the use case.